Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -374,6 +374,45 @@ namespace O3DE::ProjectManager
         }
     }
 
+
+    void GemItemDelegate::DrawPlatformText(QPainter* painter, const QRect& contentRect, const QFont& standardFont, [[maybe_unused]] const QModelIndex& modelIndex) const
+    {
+        const GemInfo::Platforms platforms = GemModel::GetPlatforms(modelIndex);
+        
+        auto xbounds = CalcColumnXBounds(HeaderOrder::Name);
+        int startX = s_platformTextleftMarginCorrection + xbounds.first;
+        QFont platformFont(standardFont);
+        platformFont.setPixelSize(s_featureTagFontSize);
+        platformFont.setBold(false);
+        painter->setFont(platformFont);
+        QString platformText = "";
+
+        
+        //UX prefers that we show platforms in reverse alphabetical order
+        for(int i = GemInfo::NumPlatforms-1; i >= 0; i--)
+        {
+            const GemInfo::Platform platform = static_cast<GemInfo::Platform>(1 << i);
+            if (platforms & platform)
+            {
+                QString singlePlatformText = GemInfo::GetPlatformString(platform);
```

Comment:
compiler is probably smart enough to avoid the copy here, but you can probably get away with something like this
```suggestion
                const QString& singlePlatformText = GemInfo::GetPlatformString(platform);
```